module/VuFind/src/VuFind/View/Helper/Root/Ratings.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/**
+ * Ratings view helper
+ *
+ * PHP version 8
+ *
+ * Copyright (C) Villanova University 2024.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * @category VuFind
+ * @package  View_Helpers
+ * @author   Demian Katz <demian.katz@villanova.edu>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/wiki/development Wiki
+ */
+
+namespace VuFind\View\Helper\Root;
+
+use VuFind\Ratings\RatingsService;
+
+/**
+ * Ratings view helper
+ *
+ * @category VuFind
+ * @package  View_Helpers
+ * @author   Demian Katz <demian.katz@villanova.edu>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/wiki/development Wiki
+ */
+class Ratings extends \Laminas\View\Helper\AbstractHelper
+{
+    /**
+     * Config constructor.
+     *
+     * @param RatingsService $service Ratings service
+     */
+    public function __construct(protected RatingsService $service)
+    {
+    }
+
+    /**
+     * Default method -- return the service.
+     *
+     * @return RatingsService
+     */
+    public function __invoke()
+    {
+        return $this->service;
```

Comment:
Creating a wrapper helper to expose the ratings service to the views is probably not a great design, but this falls in the category of "I don't want to change too many things all at once." I at least think it's better to have an explicit helper doing this work rather than running business logic through the record drivers, so it's a step in the right direction.